text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+1. Is it difficult to discover the option?
+1. What is the maintenance cost? *For example, a nice question to add could be something like, Do you want to use a CSS preprocessor? But, because Ember does not deal with SASS, LESS... by default, the interactive workflow will have to install an addon (like `ember-cli-less`) which would imply that the CLI team will have to ensure it never fails.* 
+1. Does the question affect another one? *For example, the LTS question can affect the CI file generated by the CI question (setup tests on CI with `ember-try`). Questions that modify the result of other questions too much should be avoided to prevent high maintenance.*
+
+### Triggering the Workflow
+
+The following commands, and reasonable combinations thereof, should work to enter into the interactive workflow: 
+
+1. `ember new`
+1. `ember new --interactive`
+1. `ember new -i`
+1. `ember new my-app -i`
+
+### App/Addon 
+
+Starting a new Ember app or addon as we do now (at the time of this writing) will not change. If a user types `ember new my-app` they will still get a new Ember app generated in the usual ways. However, if the user types `ember new` they will enter into an interactive workflow that will ask them a few key questions about their application. 
+
+If a user were to type in existing flags that are also questions in the interactive workflow (i.e., `ember new my-app --lang en-US --interactive`), it will skip the question entirely. 
+
+When a user types `ember new` into their command line, they will be asked these questions (defaults indicated by a `>`):
+
+- `Is this an app or an addon? (use arrow keys)`
+  - `> app`
+  - `addon`
+- `Please provide the name of your app/addon:`
+- `Please provide the spoken/content language of your app/addon: (use arrow keys)`
+  - `> en-US`
+  - `my computer's default language`
+  - `manually define a different language`
+- `Pick the package manager to use when installing dependencies:  (use arrow keys)`
+  - `> NPM`
+  - `Yarn`
+  - `ignore/skip`
+- `What CI system do you want to use? (use arrow keys)`
+  - `> travis-ci`
+  - `ignore/skip`
+
+#### App/Addon Name
+There will be no default for this question. The user must enter a response. 
+
+#### App/Addon Language
+
+- the default would be `en-US`
+- if your system has a different language set (which can be confirmed by using the `echo $LANG` command in the terminal), then _that_ language would be shown as the second option
+- if your system was already set to the default language, the second option would not be shown. 
+- for "manually define a language", we will validate the response against the allowed language codes
+- we have not added a "skip" option here, because the purpose is to focus on improved accessibility in Ember apps.
+
+#### Package Manager
+
+- ignore/skip has been added to cover the use case where the developer is in a workspace 
+
+#### CI Option
+
+Separate RFCs should further define more options for CI. 
```

Comment:
Also please take into consideration of auto-detection and a possible skip for such a question. Only if there is confidence in detection, anyway present highest probabilty as default.